testdata/book/multiple_include_a.yml

```diff
@@ -0,0 +1,18 @@
+desc: For multiple include test
+vars:
+  foo: 123
+  bar: "123-abc"
+  baz: "-23"
+  qux: 4
+  quux: "2"
+steps:
+  b:
+    include:
+      path: multiple_include_b.yml
+      vars:
+        foo: '{{ vars.foo }}'
+        bar: '{{ vars.foo }}-{{ vars.bar }}'
+        baz: '-{{ vars.foo - len(vars.bar) }}'
+        qux: '{{ len(vars.bar) - vars.qux }}'
+        quux: '{{ string(int(vars.quux) + vars.qux) }}'
+    test: vars.foo == 123 && vars.bar == "123-abc" && vars.baz == "-23" && vars.qux == 4 && vars.quux == "2"
```

Comment:
Ideally, we would like `vars.quux == 2` instead of `vars.quux == "2"` ( I understand how difficult it is ).